xpsparser/__main__.py

```diff
@@ -22,7 +22,8 @@
 
 from nomad.utils import configure_logging
 from nomad.datamodel import EntryArchive
-from xpsparser import XPSParser
+from parser import XPSParser
+#from xpsparser import XPSParser
```

Comment:
This comment does not add any value. Please remove it. Don't commit commented code, it will stay forever.